docs/section11/11-3.md

```diff
@@ -0,0 +1,60 @@
+# Service_Water_Heating - Rule 11-3
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-3
+
+**Rule Description:** "Where no service water-heating system exists or has been designed and submitted with design documents but the building will have service water-heating loads, a service water-heating system shall be modeled that matches the system type in the baseline building design"
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, proposed column, c
+
+**Evaluation Context:** P-RMD each SHW type
```

Comment:
It seems weird to evaluate this for each SWH Building Area Type. I would expect to either a single evaluation for the whole project or have it be evaluated per SWH system. If there is a requirement for 1 baseline SWH system per BAT then that seems like it would be its own rule, not incorporated here.

Reply:
It may be better to do by ServiceWaterHeatingDistributionSystem since this would encompass a SHW system as opposed to SHWHeatingEq where there could be mulitple SHWHeatingEq IDs per system. So it would be {"ServiceWaterHeatingDistributionSystem":{"SHWHeatingEq":["shwe1","shwe2"], "Building area types served":["Dormitory","Multifamlily"], "Pumps":["p1"], "Tanks":["t1"], "Piping":["piping1"], "SolarThermal":[], "Service_water_heating_uses":["swhuse1","shwuse2"]}} so all this information would be returned for each SHWHeatingEq ID.

Reply:
this is now done by SWH distribution system